apps/condo/cypress.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Is it possible to move it inside cypress folder

Reply:
Move to cypress related dir